test_files/arrow_fn.untyped/arrow_fn.untyped.js

```diff
@@ -1 +1,5 @@
-goog.module('test_files.arrow_fn.untyped.arrow_fn.untyped');var module = module || {id: 'test_files/arrow_fn.untyped/arrow_fn.untyped.js'};var /** @type {?} */ fn3 = (a) => 12;
+goog.module('test_files.arrow_fn.untyped.arrow_fn.untyped');var module = module || {id: 'test_files/arrow_fn.untyped/arrow_fn.untyped.js'};/**
```

Comment:
I think now that we presumably have working source maps, this should be fine. I'm not sure if we properly compose them in dev mode though?